docs/ManualClusterSetup.md

```diff
@@ -37,12 +37,21 @@ The [example script][example-commands] assumes that your ordinary user account h
 
 ## Getting Calico Binaries
 
-Retrieve the `calicoctl` binary.  On each host run
+Get the calico binary onto each host. It's usually safe to just grab the latest [beta](http://projectcalico.org/latest/calicoctl).  On each host run
 
-    wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.2/calicoctl
-    chmod +x calicoctl
+    wget http://projectcalico.org/latest/calicoctl
+	chmod +x calicoctl
 
-Finally, preload the Calico Docker image.  This will make the demo more responsive the first time you run it.
+The beta is not HA so may not be suitable for production environments.  Alternatively, you can download a specific [release](https://github.com/Metaswitch/calico-docker/releases/).  For example, to retrieve the most recent release, on each host run
```

Comment:
OK will update. 

Regarding the latter point, I was thinking that our release process would still update the versions in these documents, but in hindsight I'm not sure that was a good decision on my part.